src/components/Comments.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../data";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [IsLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getComments(article_id).then((commentsAPI) => {
+      setComments(commentsAPI);
+      setIsLoading(false);
+    });
+  }, [article_id]);
+
+  if (IsLoading) {
+    return <h1 className="loading-screen">comments loading...</h1>;
+  }
+
+  if (!comments) {
```

Comment:
I can't find an article with no comments to test this - but if there are no comments on this article, comments would be an empty array, and an empty array is not falsy (so `!comments` would evaluate to `false`)
Instead you could check the length of the comments array